luxonis_ml/data/parsers/coco_parser.py

```diff
@@ -241,17 +241,10 @@ def generator() -> DatasetIterator:
                     if ann.get("iscrowd", True):
                         continue
                     class_name = categories[ann["category_id"]]
-                    yield {
-                        "file": path,
-                        "annotation": {
-                            "type": "classification",
-                            "class": class_name,
-                        },
-                    }
 
                     seg = ann["segmentation"]
                     if isinstance(seg, list):
```

Comment:
We should also merge segmentation labels into unified "record" dict and yield together with other labels (class, bbox and kpts) otherwise now we can't parse COCO to include all 4 type of labels

Reply:
The COCO is still parseable. The reason it's not in the same record is because the segmentation masks are not instance segmentation, but rather one mask for all the people. So it's assigned its own task instead of it being added to `detection` task.

Reply:
The label keys would be 
```python
[
    'segmentation/segmentation',
    'segmentation/classification',
    'detection/boundingbox',
    'detection/keypoints',
    'detection/classification'
]
```

Slight annoyance is the duplicate `classification` labels for both tasks, but I'm not sure if we can easily avoid it as the two tasks can indeed have completely different classes. We just know they don't in this case.

Reply:
Actually I just figured out how we can easily fix this. We just need to explicitly set the `"task"` to the same value for both the segmentation and detection records.